src/elements/content-sidebar/SidebarNav.js

```diff
@@ -60,8 +61,26 @@ type Props = {
     signSidebarProps: SignSidebarProps,
 };
 
+const renderNavButton = (config, handleSidebarNavButtonClick) => (
```

Comment:
generally we'd like to keep one component per file. either this could be a separate component or moved to a utility file